src/redist/targets/GenerateLayout.targets

```diff
@@ -336,6 +329,13 @@
         <DownloadFileName>$(WinFormsAndWpfSharedFxArchiveFileName)</DownloadFileName>
       </BundledLayoutComponent>
 
+      <BundledLayoutComponent Include="CombinedSharedHostAndFrameworkArchive">
+        <BaseUrl>$(CoreSetupRootUrl)$(CoreSetupBlobVersion)</BaseUrl>
+        <BaseUrl Condition=" '$(DotNetBuildFromSource)' == 'true' AND '$(DotNetBuildFromSourceFlavor)' == 'Product' ">$(CoreSetupRootUrl)</BaseUrl>
+        <DownloadFileName>$(CombinedFrameworkHostArchiveFileName)</DownloadFileName>
+        <RelativeLayoutPath></RelativeLayoutPath>
+      </BundledLayoutComponent>
+
```

Comment:
We should move this into its own `<ItemGroup>` that doesn't have the `'$(IncludeWpfAndWinForms)' != 'false'` condition (so probably a new `ItemGroup` on line 345).

Reply:
Good catch. I think it should be fixed now